src/main/java/io/mosip/digitalcard/service/impl/PDFCardServiceImpl.java

```diff
@@ -159,6 +162,9 @@ public byte[] generateCard(org.json.JSONObject decryptedCredentialJson, String c
 				attributes.put("isPhotoSet",isPhotoSet);
 			}
 			uin = decryptedCredentialJson.getString("UIN");
+			if(additionalAttributes.containsKey("template")) {
+				template = additionalAttributes.get("template").toString();
```

Comment:
Look like this variable has the template filename, lets get a template code in the event and maintain the filename mapping separately in the digital card service property files

Reply:
taken care